### PR TITLE
Add keylime_policy-commands test

### PR DESCRIPTION
**Pull request by ansasaki**
Test policy generation tool 'keylime_policy' with all available options

### CODE REVIEW

**Review by kkaarreell — APPROVED** [review]
Thank you for the test

**Comment by kkaarreell** [comment]
Please rebase.

File: functional/keylime_policy-commands/test.sh [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        rlRun "keylime_policy create runtime -h"
+        rlRun "keylime_policy create runtime --help"
+    rlPhaseEnd
+
+    # Generate runtime policy from filesystem
+
+    rlPhaseStartTest "Include the IMA log with --use-ima-measurement-list"
+        # TODO: Currently, the output is not parseable as JSON directly with a pipe.
+        # Possibly related to https://github.com/keylime/keylime/issues/1613
+        # rlRun -s "keylime_policy create runtime --use-ima-measurement-list | jq '.digests'"
```
[the remark this answers is not in the document]
Comment (ansasaki) [reply functional/keylime_policy-commands/test.sh:65]: I'll rebase and make the changes to use `--output`

**Pull request merged**